tests/test_utils/tests.py

```diff
@@ -263,6 +263,43 @@ def make_configuration_query():
                 list(Car.objects.all())
 
 
+class MultiDBRouter:
+    def allow_relation(self, obj1, obj2, **hints):
+        return True
+
+    def db_for_read(self, model, **hints):
+        if model == Car:
+            return "other"
+        return "default"
+
+    db_for_write = db_for_read
+
+
+@override_settings(
+    DATABASE_ROUTERS=["%s.MultiDBRouter" % __name__],
```

Comment:
use an f-string

Reply:
I was copying the style from another test. I'll change it.